turtlebot3_fake_node/include/turtlebot3_fake_node/turtlebot3_fake_node.hpp

```diff
@@ -72,9 +74,9 @@ class Turtlebot3Fake : public rclcpp::Node
   // Function prototypes
   void init_parameters();
   void init_variables();
-  void command_velocity_callback(const geometry_msgs::msg::Twist::SharedPtr cmd_vel_msg);
+  void cmd_vel_callback(const geometry_msgs::msg::Twist::SharedPtr cmd_vel_msg);
   void update_callback();
-  bool update_odometry(const rclcpp::Duration & diff_time);
+  bool update_odom(const rclcpp::Duration & diff_time);
```

Comment:
Why you change this function name?

Reply:
There are no specific reasons for it. I just thought it would improve code readability since cmd_vel and odom are often used in other tb3 packages.

Reply:
The former is correct. Please use full naming.